src/cow-react/modules/swap/containers/EthFlowStepper/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { EthFlowStepper as Pure, SmartOrderStatus } from '@cow/modules/swap/pure/EthFlow/EthFlowStepper'
+import { useDetectNativeToken } from '@cow/modules/swap/hooks/useDetectNativeToken'
+import { Order } from 'state/orders/actions'
+import { NATIVE_CURRENCY_BUY_ADDRESS } from 'constants/index'
+
+type Props = {
+  storeOrder: Order | undefined
+}
+
+export function EthFlowStepper(props: Props) {
+  const { storeOrder } = props
+  const { native } = useDetectNativeToken()
+
+  const state = mapOrderToEthFlowStepperState(storeOrder)
+  const isEthFlowOrder = getIsEthFlowOrder(storeOrder)
+
+  if (!storeOrder || !state || !isEthFlowOrder) {
+    return null
+  }
+
+  const nativeTokenSymbol = native.symbol as string
+  const tokenLabel = storeOrder.outputToken.symbol as string
+
+  const order = {
+    // The creation hash is only available in the device where the order is placed
+    createOrderTx: storeOrder.orderCreationHash || '',
+    orderId: storeOrder.id,
+    state,
+    isExpired: storeOrder.status === 'expired',
+    // rejectedReason?: TODO: address when dealing with rejections
+  }
+
+  // TODO: fill these in when dealing with rejections
+  const refund = {
+    // refundTx?: string
+    isRefunded: false,
+  }
+
+  // TODO: fill these in when dealing with cancellations
+  const cancelation = {
+    // cancelationTx?: string
+    isCanceled: false,
+  }
+
+  return (
+    <Pure
+      nativeTokenSymbol={nativeTokenSymbol}
+      tokenLabel={tokenLabel}
+      order={order}
+      refund={refund}
+      cancelation={cancelation}
+    />
+  )
+}
+
+function mapOrderToEthFlowStepperState(order: Order | undefined): SmartOrderStatus | undefined {
+  // NOTE: not returning `CREATED` as we currently don't track the initial tx execution
+  if (!order) {
+    return
+  } else if (order.status === 'creating') {
```

Comment:
Very small nitpick: `else` is optional in the case when you immediately return value

Reply:
Agree!
I just thought did it because I can save one line and put `else if` in the same line 😅 

I don't mind changing it if you think this isn't a strong reason to keep it